.github/workflows/build-test.yml

```diff
@@ -129,3 +129,32 @@ jobs:
                 CLOUD_PROVIDER: ${{ matrix.cloud }}
                 TARGET_DOCKER_TEST_IMAGE: ${{ matrix.image }}
               run: ./ci/test.sh
+    test-ubuntu:
+      needs: build
+      name: Tests on Ubuntu
+      runs-on: ubuntu-latest
+      strategy:
+        fail-fast: false
+        matrix:
+          cloud: [ 'AWS', 'AZURE', 'GCP' ]
+          nodeVersion: ['18.x', '20.x']
```

Comment:
don't we want for now to also add 14.x and 16.x?

Reply:
We have tested all versions on linux - centos7. It me opinion it is enough. It is a matrix and every next node version run 3 new test processes (for every cloud). We have plans to drop support for versions <1 8